src/ModularPipelines/ModularPipelines.csproj

```diff
@@ -44,10 +44,5 @@
       <PackageReference Include="YamlDotNet" Version="15.1.0" />
     </ItemGroup>
 
-    <ItemGroup>
```

Comment:
Hmm I don't want to remove this line. This makes dotnet pack aware that this is a dependency, so that when you install the base package, the analyzer is pulled in as well.

What issue are you getting by leaving this in?

If it is giving an issue locally, what we can do is add a condition to the item group so that this is brought in if we're in the context of the build pipeline (which is where the nuget packages are built and published.)

Reply:
If so, the condition on the item group would be this:

Condition="'$(GITHUB_ACTIONS)' == 'true'"

This should mean it's brought in when building (and so makes it a dependency that will be pulled in) - but won't be pulled in locally, as that env variable will be null

Reply:
Sorry, yes. These are just changes to be able to build locally. 

The actual change for .NET8 to work (and it minor) is here:
https://github.com/thomhurst/ModularPipelines/pull/414/commits/55a702bc48dac408b1cf02f25de8877dbb247f3c

Reply:
What's the error you're getting out of curiosity?